### DISCUSSION TITLE
ToolBox CD switcher

### DISCUSSION THREAD

**Discussion by tjjq44** [BlueSCSI/BlueSCSI-v2#190] [Ideas]
For people using "CDx" folder with multiple ISOs inside, is it possible to add a "manual CD mount" option in "CD switcher" in order to avoid automatic "next CD insertion" when current one is dragged to trash ?

**Comment by tjjq44** [comment]
And ideally, make the thing remember if there was no CD inserted on last use to keep no CD inserted on next start of the computer.

**Reply by jimfrob** [reply]
This is almost exactly what I was thinking of suggesting. It would make the behavior of my System 7 desktop much better as booting with an empty CD drive wouldn't screw up the desktop icon positions.

**Reply by erichelgeson** [reply]
@jimfrob not sure if you got notified as i replied to the parent but see the reply below.

**Comment by erichelgeson** [comment]
There is a `StartEjected` option now, eg, if you have a folder `CD3` in `bluescsi.ini` add

```ini
[SCSI3]
StartEjected=true
```
